### DISCUSSION TITLE
Seconda batteria di domande al professore

### DISCUSSION THREAD

**Discussion by AccaEmme** [AccaEmme/CARE#4] [Q&A]
1) Hermann: in un approccio di tipo MVC, può essere corretto definire come "Model" le classi e come "Control" il MySQLDataManagement e DataManagement?

**Comment by DonyBerserker** [comment]
2. Leggendo i capitoli riguardanti l'architettura, come devono essere preparati i disegni per la documentazione (Cap 6 & 7)?

**Comment by DonyBerserker** [comment]
3. Nella documentazione riguardante l'architettura dopo aver spiegato le motivazioni riguardo la scelta di un tipo di sistema, va anche inserita un'immagine del sistema ideato, oppure si passa direttamente alla spiegazione dei sottosistemi?

**Reply by AccaEmme** [reply]
4. Nel caso della gestione delle eccezioni, posso considerare "blackbox" un comportamento di errore? Ad esempio la divisione con zero al denominatore deve restituire un qualcosa di errore, una eccezione. Il fatto che venga generata una eccezione questa può essere definita "blackbox" o rientra nella tipologia "white box" perché la gestione del comportamento anomalo è stata meglio definita come "eccezione" (e non diversamente)?

**Comment by follen99** [comment]
5. Vista la necessità di dover sviluppare un'applicazione lato web per poter gestire il sistema, sarebbe possibile dedicare una pillola a questo topic?
